package.json

```diff
@@ -59,6 +59,15 @@
   "main": "app.js",
   "private": true,
   "scripts": {
+    "docker:build": "docker compose -f ./docker/docker-compose.yml build --no-cache",
+    "docker:clean:containers": "docker rm $(docker ps -qa --no-trunc --filter \"status=exited\") || true",
+    "docker:clean:images": "docker rmi $(docker images --filter \"dangling=true\" -q --no-trunc) || true",
+    "docker:clean:volumes": "docker volume rm $(docker volume ls -qf dangling=true) || true",
+    "docker:clean": "npm run docker:clean:containers;npm run docker:clean:images;npm run docker:clean:volumes",
+    "docker:down": "docker stop $(docker ps -q)",
+    "docker:nuke": "docker rmi $(docker images -f=\"reference=cuttle*\" -q --no-trunc) || true; npm run docker:clean",
```

Comment:
How is this one different from `clean`?

Reply:
i.e. even kill things that are actively running?

Reply:
Nah, you'd still need to stop them as far as I understand. We could maybe preface this with a docker down but I think we shouldn't do that for safety as a final barrier lol.